src/app/_components/Selector/index.tsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+
+import { RefObject } from 'react';
+
+import Icon from '@/src/components/Icon';
+
+import useDropdown from './useDropdown';
+
+interface Item {
+  text: string;
+}
+
+interface SelectorProps {
+  items: Item[];
+  placeholder: string;
+}
+
+const Selector = ({ items, placeholder }: SelectorProps) => {
+  const {
+    isOpen,
+    toggleDropdown,
+    selectedItem,
+    updateSelectedItem,
+    dropdownRef,
+  } = useDropdown<Item>();
+
+  return (
+    <div
+      className='relative inline-block'
+      onClick={toggleDropdown}
+      ref={dropdownRef as RefObject<HTMLDivElement>}
+    >
+      <button className='text-bold flex w-full justify-between rounded border bg-gray-accent1 p-2'>
```

Comment:
```suggestion
      <button className='text-bold flex w-full justify-between rounded border bg-gray-accent1 p-2 items-center'>
```
이렇게 해서 규격을 조금 잘 잡아주면 좋을 것 같아요.